experiments/react-native-uifabric/src/components/RadioGroup/RadioButton.settings.ts

```diff
@@ -0,0 +1,86 @@
+import { radioButtonName, IRadioButtonType } from './RadioButton.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import { ViewProps } from 'react-native';
+
+export const settings: IComposeSettings<IRadioButtonType> = [
+  {
+    root: {
+      accessible: true,
+      acceptsKeyboardFocus: true,
+      accessibilityRole: 'radio',
+      style: {
+        display: 'flex',
+        alignItems: 'flex-start',
+        flexDirection: 'row',
+        alignSelf: 'flex-start',
```

Comment:
we should probably be more selective with when we set alignSelf. alignItems on the container in the group should be sufficient.

Reply:
Removed